cpp/tests/streams/lists_test.cpp

```diff
@@ -167,6 +169,7 @@ TEST_F(ListTest, Distinct)
   cudf::lists::distinct(list_col,
                         cudf::null_equality::EQUAL,
                         cudf::nan_equality::ALL_EQUAL,
+                        cudf::duplicate_keep_option::KEEP_FIRST,
```

Comment:
Why is this `KEEP_FIRST`? I would've expected it to be `KEEP_ANY` like in the default. Is the default incorrect?